src/com/jaquadro/minecraft/storagedrawers/core/ModItems.java

```diff
@@ -50,6 +53,9 @@ public void init () {
             GameRegistry.registerItem(personalKey, "personalKey");
         if (StorageDrawers.config.cache.enableTape)
             GameRegistry.registerItem(tape, "tape");
+        if (true) {
```

Comment:
rm

Reply:
It's a placeholder for an actual config check

Reply:
Implement the config check or remove it :P

Reply:
Yessir :(